torch/onnx/_internal/torchscript.py

```diff
@@ -0,0 +1,426 @@
+"""Utilities for manipulating the torch.Graph object and the torchscript."""
```

Comment:
`torchscript.py` is too generic. Torch Script can refer to the TorchScript "language", `torch.jit.script` engine or maybe TorchScript IR.

`_scalar` and `_is_caffe2_aten_fallback` don't depend/implement on anything related to TorchScript in any of the context above, for example. Instead, they process `torch.Tensor` properties or globals defined at `torch.onnx` namespace

If we want to mix torch script and helpers in a single file, `utils.py` still looks a good name. Or we split files to keep different concepts decoupled

In fact, aren't some of the helpers below not related to torch script? Examples would be `class GraphLike` which is used by

Reply:
> `torchscript.py` is too generic. Torch Script can refer to the TorchScript "language", `torch.jit.script` engine or maybe TorchScript IR.

I agree. Any naming suggestions?

> `_scalar` and `_is_caffe2_aten_fallback` don't depend/implement on anything related to TorchScript in any of the context above, for example. Instead, they process `torch.Tensor` properties or globals defined at `torch.onnx` namespace

These are private helper functions that we need to implement the public methods.